src/aosm/azext_aosm/build_processors/nfd_processor.py

```diff
@@ -96,10 +98,26 @@ def get_artifact_details(
             )
             and self.input_artifact.network_function_definition.properties.network_function_template
         ):
-            params = {
-                "nfvi_type":
-                self.input_artifact.network_function_definition.properties.network_function_template.nfvi_type
-            }
+            # Split for line-too-long linting errors
+            nf_type = self.input_artifact.network_function_definition.properties.network_function_type
```

Comment:
should we be doing this in a try, except? Is it possible for us to get a NFDV that doesn't have these properties?

Reply:
there is alradyt a check for network function template above, and since this is a get from azure, there should laways be an nf type